src/main/java/com/blamejared/crafttweaker/api/item/IItemStack.java

```diff
@@ -46,7 +47,7 @@
      */
     @ZenCodeType.Getter("registryName")
     default String getRegistryName() {
-        return getInternal().getItem().getRegistryName().toString();
+        return String.valueOf(getInternal().getItem().getRegistryName());
```

Comment:
If this PR already is a breaking change because of tag stuff, does it make sense to change this to an MCResourceLocation, see: https://discord.com/channels/136877912698650625/514195779120922624/774743729017454623